packages/react/src/components/form/Switch/Switch.tsx

```diff
@@ -0,0 +1,123 @@
+import type { InputHTMLAttributes, ReactNode, SVGAttributes } from 'react';
+import React, { forwardRef } from 'react';
+import cn from 'classnames';
+import { PadlockLockedFillIcon } from '@navikt/aksel-icons';
+
+import { omit } from '../../../utils';
+import { Label, Paragraph } from '../../Typography';
+import type { FormFieldProps } from '../useFormField';
+
+import classes from './Switch.module.css';
+import { useSwitch } from './useSwitch';
+
+const SwitchIcon = (props: SVGAttributes<SVGElement>) => (
```

Comment:
Jeg tror fortsatt jeg heller mot å ha denne i en egen fil. Jeg ser at den da tar opp ekstra plass blant filene, men nå tar den opp ekstra plass i komponentkoden. Personlig foretrekker jeg også å holde det visuelle separert fra det funksjonelle så langt det lar seg gjøre. Men dette blir jo en smakssak.

Reply:
Ja, denne ble litt større. Vi kan sikkert gjøre det for alle tre komponentene, slik at det er likt, og ikke minst med `Helptext` :)